tutorials/napps/development_environment_setup.rst

```diff
@@ -173,17 +173,59 @@ and run:
     explained, as well as NApp management, in the next tutorials.
 
 Now that Kytos is running, switch back to the previous window and install the
-NApps using the ``kytos`` command line utility. You will also disable the NApps,
-just for now.
+NApps using the github command line utility.
+
+of_core:
+
+.. code-block:: console
+
+  $ git clone https://github.com/kytos-ng/of_core.git
+  $ cd of_core
+  $ python setup.py develop
+
+flow_manager:
+
+before installing flow_manager you must installl storehouse
 
 .. code-block:: console
 
-  $ kytos napps install kytos/of_core \
-     kytos/flow_manager \
-     kytos/topology \
-     kytos/of_l2ls \
-     kytos/of_lldp
+  $ git clone https://github.com/kytos-ng/storehouse.git
+  $ cd storehouse
+  $ python setup.py develop
+
+  #now install flow_manager
+
+  $ git clone https://github.com/kytos-ng/flow_manager.git
+  $ cd flow_manager
+  $ python setup.py develop
+
+topology:
+
+.. code-block:: console
 
+  $ git clone https://github.com/kytos-ng/topology.git
+  $ cd topology
+  $ python setup.py develop
+
+of_lldp:
+
+.. code-block:: console
+
+  $ git clone https://github.com/kytos-ng/of_lldp.git
+  $ cd of_lldp
+  $ python setup.py develop
+
+of_l2ls:
+
+.. code-block:: console
+
+  $ git clone https://github.com/kytos-ng/of_l2ls.git
+  $ cd of_l2ls
+  $ python setup.py develop
+
+now disable all napps
+
+.. code-block:: console
```

Comment:
@usmanazFIU could you also find and adapt the mentions about Python 3.6 and higher? At the moment, our team is officially supporting Python 3.9, so users should be installing and using Python 3.9. (3.10 isn't supported yet either). 

Thanks.

Reply:
sure

Reply:
@viniarck where should I mention it. at the top of the document may be?

Reply:
The places were you replaced with 3.9 look great.